Content.Server/Radio/Components/RadioSpeakerComponent.cs

```diff
@@ -24,4 +26,7 @@ public sealed partial class RadioSpeakerComponent : Component
 
     [DataField("enabled")]
     public bool Enabled;
+
+    [DataField("ChatType", customTypeSerializer: typeof(ByteSerializer))]
+    public byte ChatType = 2;
```

Comment:
```suggestion
    [DataField]
    public InGameICChatType ChatType = InGameICChatType.Whisper;
```
- Why use a `byte` when there's a typed enum? This also means you don't need a custom serializer.
- `Datafield`'s name is the same as the implicit one generated from the field name.

Reply:
We want it to modifiable via yaml. Apparently there is no easy way to serialize from yaml to an enum nicely, at least not that I could figure out.

Reply:
never mind I was being stupid

Reply:
However when I don't have the datafield one it does break it for some reason.

Reply:
You need [Datafield], but not the name.
Note also that it converts FieldName to fieldName, so just check the caps.